updated the GRUB entry (#246)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/g.adoc

[discrete]
[[grub]]
==== image:images/yes.png[yes] GRUB (noun)

> *Description*: "GRUB" is an acronym for "GRand Unified Bootloader", which is a Linux boot loader. (GRUB 2 has replaced what was formerly known as GRUB, i.e. version 0.9x, which has, in turn, become GRUB Legacy.)

Reviewer: I'd recommend getting rid of the parentheses around the sentence too and removing the "i.e.". The sentence is a bit complex, so my suggestion breaks it up into two sentences, but there are other solutions that could work I'm sure!

```suggestion
*Description*: "GRUB" is an acronym for "GRand Unified Bootloader", which is a Linux boot loader. Note that GRUB 2 has replaced what was formerly known as GRUB, which was version 0.9x. GRUB version 0.9x is now known as _GRUB Legacy_.
```

Author: Great, I like your version better! Could you please push it to the repo?

Reviewer: @jheves I'd recommend that you make the change manually from your terminal/editor - just in case there are any further updates that need to be made, then your local branch and this PR aren't out of sync

Author: @bergerhoffer OIC! Consider it done :)

> *Description*: "GRUB" is an acronym for "GRand Unified Bootloader", which is a Linux boot loader. (Note: GRUB 2 has replaced what was formerly known as GRUB (i.e. version 0.9x), which has, in turn, become GRUB Legacy.)

Reviewer 2: Is there a way to avoid the parentheses, and the parentheses within the parentheses? Does that sentence even need to be a Note? If not, maybe remove the `Note:` as well as the parentheses. Or if needs to be Note admonition, then use an Asciidoc Note admonition.

*Use it*: yes

> *Incorrect forms*: Grub, GRUB 2, GRUB2 (Note: GRUB 2 has replaced what was formerly known as GRUB (i.e. version 0.9x), which has, in turn, become GRUB Legacy.)

Reviewer: @jheves we usually want our incorrect forms list to be just a list of terms, and any details about why they are incorrect should go into the description field.

Would you mind moving the details about GRUB 2 replacing the former GRUB up as a sentence in the description?

Author: Sure thing! I'll do it asap! Thank you for the feedback, Andrea!
